File: core/model/src/main/java/com/mifos/mobilewallet/model/domain/NewAccount.java
package com.mifos.mobilewallet.model.domain;

import java.util.Date;

public class NewAccount {

    private int clientId;
    private String productId;
    private Date submittedOnDate;
    private String accountNo;
    private String locale;
    private String dateFormat;

    public NewAccount(int clientId, String accountNo) {
        this.clientId = clientId;
        this.accountNo = accountNo;
    }
}

File: core/model/src/main/java/com/mifos/mobilewallet/model/domain/NewAccount.kt
package com.mifos.mobilewallet.model.domain

import java.util.Date
data class NewAccount(
    var clientId: Int,
    var productId: String? = null,
    var submittedOnDate: Date? = null,
    var accountNo: String,
    var locale: String? = null,
    var dateFormat: String? = null
)
